config/views.yaml

```diff
@@ -0,0 +1,202 @@
+---
+component_readiness:
+  - name: 4.18-main
+    base_release:
+      release: "4.16"
+      relative_start: now-30d
+      relative_end: now
+    sample_release:
+      release: "4.18"
+      relative_start: now-7d
+      relative_end: now
+    variant_options:
+      column_group_by:
+        Architecture: {}
+        Network: {}
+        Platform: {}
+      db_group_by:
+        Architecture: {}
+        FeatureSet: {}
+        Installer: {}
+        Network: {}
+        Platform: {}
+        Suite: {}
+        Topology: {}
+        Upgrade: {}
+      include_variants:
```

Comment:
We can deal with this later, but can 4.17+ drop `sdn` from the include networking variants?